app/src/main/java/edu/uc/coffeens/stocktracker/ui/main/MainViewModel.kt

```diff
@@ -36,10 +34,10 @@ class MainViewModel : ViewModel() {
             .document()
             .set(stocks)
             .addOnSuccessListener {
-                Log.d("Firebase", "document saved")
+                Log.e("Firebase", "document saved")
             }
             .addOnFailureListener {
-                Log.d("Firebase", "save failed")
+                Log.e("Firebase", "save failed")
```

Comment:
updated to log.e so if these errors occur, they can be spotted in something other than debug mode.